code/datums/entities/player_sticky_ban.dm

```diff
@@ -131,3 +131,114 @@ BSQL_PROTECT_DATUM(/datum/entity/stickyban)
 		"ip",
 		"linked_stickyban",
 	)
+
+/proc/get_player_is_alt(ckey)
+	var/list/datum/view_record/known_alt/alts = DB_VIEW(/datum/view_record/known_alt, DB_COMP("ckey", DB_EQUALS, ckey))
+	if(!length(alts))
+		return FALSE
+
+	var/ckeys = list()
+	for(var/datum/view_record/known_alt/alt as anything in alts)
+		ckeys += alt.player_ckey
+
+	return ckeys
+
+/client/proc/add_known_alt()
+	set name = "Add Known Alt"
+	set category = "Admin.Alt"
+
+	var/player_ckey = ckey(tgui_input_text(src, "What is the player's primary Ckey?", "Player Ckey"))
+	if(!player_ckey)
+		return
+
+	var/datum/entity/player/player = get_player_from_key(player_ckey)
+	if(!istype(player))
+		return
+
+	var/existing_alts = get_player_is_alt(player_ckey)
+	if(existing_alts)
+		var/confirm = tgui_alert(src, "Primary Ckey [player_ckey] is already an alt for [english_list(existing_alts)].", "Primary Ckey", list("Confirm", "Cancel"))
+
+		if(confirm != "Confirm")
+			return
+
+	var/whitelist_to_add = ckey(tgui_input_text(src, "What is the Ckey that should be added to known alts?", "Alt Ckey"))
+	if(!whitelist_to_add)
+		return
```

Comment:
The same alt can be added successfully to the same ckey multiple times: 
![image](https://github.com/user-attachments/assets/2bd29e90-69e0-4cc5-ac07-b5e3b062ba80)
Its recoverable from, but you'll have to repeat the removal process for each.